lib/index.js

```diff
@@ -145,17 +145,17 @@ api.insert = brCallbackify(async ({actor, account, meta}) => {
       roles[i] = api.generateResource({role, id: account.id});
     } else if(role.generateResource) {
       // unknown generation directive
-      throw new BedrockError(
-        'Could not create Account; unknown ResourceRole rule.',
+      throw new
+      BedrockError('Could not create Account; unknown ResourceRole rule.',
         'NotSupportedError', {sysResourceRole: role});
```

Comment:
Yeah, this isn't good. We need to figure out the right eslint rules so this is ok -- or if that's not possible, disable whatever is causing this to get messed up.

Reply:
I recommend disable today, reformat to pretty code (or use prettier), then crowd source the right rules.

Reply:
oh yeah we should probably collect a file with all the oddities of our func-paren usage and make sure whatever plugin is choosen it works in every case.